src/DeploymentProvider/Providers/PublishProfileWebAppContainerDeploymentProvider.ts

```diff
@@ -0,0 +1,12 @@
+import { BaseWebAppDeploymentProvider } from './BaseWebAppDeploymentProvider';
+
+export class PublishProfileWebAppContainerDeploymentProvider extends BaseWebAppDeploymentProvider {
+    
+    public async DeployWebAppStep() {
+        let appName: string = this.actionParams.appName;
+        let images: string = this.actionParams.images;
+        // TODO - how to check if its a linux container
```

Comment:
Get the web app details and check if there is any property which indicates if it is a linux type.

Reply:
Working on getting web app details using publish profile details.